CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## v1.6.0
+
+This release focuses on security enhancements. It provides additional isolators, creating a new mount namespace per app. Also a new version of CoreOS 1032.0.0 with systemd v229 is being used in stage1.
+
+#### New features and UX changes
+
+- stage1: implement read-only rootfs ([#2624](https://github.com/coreos/rkt/pull/2624)). Using the Pod manifest readOnlyRootFS option mounts the rootfs of the app as read-only using systemd-exec unit option ReadOnlyDirectories.
```

Comment:
"Pod manifest readOnlyRootFS"

Should you add a link to the relevant part of appc/spec for this feature? So users know how to use this.